pkg/policy/policy.go

```diff
@@ -78,21 +95,50 @@ func (pp Policies) All() []string {
 	return policyNames
 }
 
-func (pp Policies) Properties() map[string]interface{} {
-	usedCustom := 0
-	policies := make([]*Meta, 0)
+func (pp Policies) Get(policyName, subPath string) Policies {
```

Comment:
[non blocking] What do you think about returning a single Policy? Both places we call `Get` checks for `len > 0`. We could return a single policy pointer and check for `nil` instead.
Also the function always returns a single policy so it will better reflect what it does.

Reply:
Will change this